scripts/update-sdk-version.js

```diff
@@ -1,28 +1,26 @@
 const path = require('path');
 const fs = require('fs');
+
+const { currentVersion, getLatestNPMJSSdkVersion }  = require('./version-utils');
```

Comment:
Does not look like we are using `getLatestNPMJSSdkVersion` below. Do we need this `update-sdk-version` script?